doc/ref_model/chapters/chapter03.md

```diff
@@ -543,33 +543,52 @@ SyncE architecture minimally requires replacement of the internal clock of the E
 * Automation: Enable end-to-end automation, from Physical Storage installation and provisioning to automation of workloads (VNF/CNF) onboarding.
 * Openness: All storage is based on open source or standardized APIs (North Bound Interfaces (NBI) and South Bound Interfaces (SBI)) and should enable integration of storage components such as Software Defined Storage controllers.
 * Scalability: Storage model enables scalability to enable small up to large deployments.
 * Workload agnostic: Storage model can provide storage functionality to any type of workloads, including: tenant VNF, CNF and Infrastructure Managment whether this is via BareMetal or Virtualised Deployments.
-* Operationally Ammenable: The storage must be ammenable to consistent set of operational processes for: Non-Disruptive Capacity Expansion and Contraction, Backup/Restoration and Archive and Performance Management. Where applicable (Backup/Restoration/Archive) these processes should also be able to be provided to tenants for there own delegated management.
+* Operationally Ammenable: The storage must be ammenable to consistent set of operational processes for: Non-Disruptive Capacity Expansion and Contraction, Backup/Restoration and Archive and Performance Management. Where applicable (examples are: Backup/Restoration/Archive) these processes should also be able to be provided to tenants for their own delegated management.
+* Security Policy Ammenable: The storage sub-systems must be ammenable to policy based security controls covering ares such as: Encryption for Data at Rest / In Flight, Delegated Tenant Security Policy Management, Platform Management Security Policy Override, Secure Erase on Device Removal and others
 * Future proof: Storage model is extendible to support known and emerging technology trends covering spectrum of memory-storage technologies including Software Defined Storage with mix of SATA- and NVMe-based SSDs, DRAM and Persistent Memory, integrated for multi-clouds, and Edge related technologies.
 
+NOTE: In a market assessemt of storage (for a "Ceph" based solution) costs done in 2019, the following comparative costs where found when trying to meet the different Storage Class needs across varies performance levels, for consistent usable capacity:
+* Ultra High Performance - for "Platform Native" case based on "Converaged Scalable Software Defined Storage" architecture where cost included: Software, Compute Chassis, NVMe Flash, 25GbE NICs: 64% > Avg
+* High Performance - for "Platform Native" and "Shared Storage" cases based on "Converged Scalable Software Defined Storage" architecture where cost included: Software, Compute Chassis, SATA Flash, 25GbE NICs: 46% > Avg
+* Value - for "Platform Native", "Shared Storage" and "Object Storage" based on "Converage Storage with Flash Cached Disks" architecture where cost included: Software, Compute Chassis, SAS Disk with SATA Flash Cache, 25GbE NICs: -12% < Avg
```

Comment:
Converage?

Reply:
Thank you, should be "converged", corrected.